osd/KubePersistentVolumeFillingUpWarn.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Warning",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: update cluster logging configuration",
+    "description": "cluster requires you to take action. Elasticsearch is close to capacity. SRE will increase capacity one time causing a period while Elasticsearch is unavailable. Old logs will be retained. Logs for pods deleted while migration is in progress may be lost. SRE recommends reducing application logging on your cluster to ensure logging continues to function smoothly in the future.",
```

Comment:
needs info to which capacity

Reply:
added percent capacity param. fortunately osdctl will not post with undefined params